src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/CABI.java

```diff
@@ -32,7 +32,8 @@
 public enum CABI {
     SysV,
     Win64,
-    AArch64;
+    AArch64,
```

Comment:
This should be Linux AArch64. Btw, should we use Arm64 instead, which seems to be the more common name to refer to the platform?

Reply:
I thought about using MacOsArm64 as that would be closer to the terminology Apple uses, but then we use AArch64 pretty consistently everywhere else in OpenJDK/Hotspot to refer to this architecture, so it seems strange to deviate here.